index.js

```diff
@@ -92,6 +94,7 @@ export default class QRCodeScanner extends Component {
       fadeInOpacity: new Animated.Value(0),
       isAuthorized: false,
       isAuthorizationChecked: false,
+      disablingByUser: false,
```

Comment:
can we give this a more descriptive/explicit name like `disableVibrationByUser`, thanks!

Reply:
@maxsdev would you please prepare change? :)